2-GenSoftware.sh

```diff
@@ -82,7 +84,7 @@ execute sudo apt-get install lm-sensors hddtemp -y
 execute sudo apt-get install psensor xsensors -y
 execute sudo apt-get update
 
-execute sudo apt-get install redshift redshift-gtk shutter -y
+execute sudo apt-get install flameshot -y
```

Comment:
Assuming `redshift` is being removed since there's a Night Light option by default since 18.04? Just a comment, should be okay

Reply:
So `flameshot` isn't there for 16.04. Should we still keep 16.04 builds? 15 builds are quite a lot, and probably only `xenial-ml-basic` will work
Or we add `if` checks before specific packages (that doesn't seem to be very useful)

Reply:
Good catch, we could just install flameshot if Ubuntu>16

Ubuntu 16's regular updates EOL is 2021 I think. And I think we aren't completely at a stage where we can remove 16 support yet, there are still a lot of Caffe codes etc that rely on the 16.04 ecosystem. Probably sometime next year?

Reply:
Hmmm but there might be quite a lot of incompatibilities, just saw the next one. Okay, I guess we can remove testing for 16.04, someone who wants to use it should always be able to go back to a previous commit of this repo and use the script version from then